extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -118,6 +118,9 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 			 */
 			if (value.selected) {
 				this.migrationStateModel._azureAccount.properties.tenants = [this.migrationStateModel.getTenant(value.index)];
+				this.migrationStateModel._subscriptions = undefined!;
```

Comment:
Shouldn't there be some other logic that goes and loads the subscriptions for the newly selected tenant? How is this getting populated otherwise?

Reply:
So, in the next page when the subscriptions are undefined we just go and fetch subscriptions for the accounts.